ttnn/cpp/ttnn/operations/moreh/moreh_group_norm_backward/moreh_group_norm_backward_pybind.cpp

```diff
@@ -0,0 +1,36 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "ttnn/operations/moreh/moreh_group_norm_backward/moreh_group_norm_backward.hpp"
+
+#include "moreh_group_norm_backward_pybind.hpp"
+#include "pybind11/cast.h"
+#include "pybind11/decorators.hpp"
+
+namespace ttnn::operations::moreh::moreh_group_norm_backward {
+    void bind_moreh_group_norm_backward_operation(py::module &module) {
+        bind_registered_operation(
+            module,
+            ttnn::moreh_group_norm_backward,
+            "Moreh moreh_group_norm_backward Operation",
+            ttnn::pybind_arguments_t{
+                py::arg("output_grad"),
+                py::arg("input"),
+                py::arg("mean"),
+                py::arg("rstd"),
+                py::arg("num_groups"),
+                py::kw_only(),
+                py::arg("are_required_outputs") = std::vector<bool>{true, false, false, false},
```

Comment:
Since the output of the backward operation is three, the vector size should also be three